test/server/document.spec.js

```diff
@@ -113,19 +96,16 @@ describe('Documents', () => {
     it('should return documents in order from the most recent', (done) => {
       request
         .get('/api/documents/')
+        .set({ 'x-access-token': token })
         .query({
-          token,
```

Comment:
**Code Quality - Nitpick**
- set the token as a header as opposed to sending it in the query